helix-core/src/test/java/org/apache/helix/integration/messaging/TestBatchMessage.java

```diff
@@ -45,13 +45,16 @@
 
 public class TestBatchMessage extends ZkTestBase {
   class TestZkChildListener implements IZkChildListener {
-    int _maxNbOfChilds = 0;
+    int _maxNumberOfChildren = 0;
 
     @Override
-    public void handleChildChange(String parentPath, List<String> currentChilds) throws Exception {
-      System.out.println(parentPath + " has " + currentChilds.size() + " messages");
-      if (currentChilds.size() > _maxNbOfChilds) {
-        _maxNbOfChilds = currentChilds.size();
+    public void handleChildChange(String parentPath, List<String> currentChildren) {
+      if (currentChildren == null) {
```

Comment:
Is this an issue in production code?

Reply:
Not sure. Haven't checked the prod logs.